functions/pdf.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * pdf function
+ *
+ * Example invocation:
+ *
+ * content({
+ *  page: await browser.newPage(),
+ *  context: {
+ *    url: 'https://example.com',
+ *    html: '<div>example</div>
+ *    options: {
+ *      ...
+ *      see https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagepdfoptions for available options
+ *    }
+ *  },
+ * });
+ *
+ * @param args - object - An object with a puppeteer page object, and context.
+ * @param args.page - object - Puppeteer's page object (from await browser.newPage)
+ * @param args.context - object - An object of parameters that the function is called with. See src/schemas.ts
+ */
+module.exports = async function content ({ page, context }) {
+  const { url, html, options } = context;
```

Comment:
Probably need to drop the console.logs below

Reply:
Woops! Didn't mean to leave those in.